pkg/kubelet/util/pluginwatcher/plugin_watcher.go

```diff
@@ -112,12 +113,12 @@ func (w *Watcher) Start() error {
 					if event.Op&fsnotify.Create == fsnotify.Create {
 						err := w.handleCreateEvent(event)
 						if err != nil {
-							glog.Errorf("error %v when handling create event: %s", err, event)
+							glog.V(5).Infof("error %v when handling create event: %s", err, event)
 						}
 					} else if event.Op&fsnotify.Remove == fsnotify.Remove {
 						err := w.handleDeleteEvent(event)
 						if err != nil {
-							glog.Errorf("error %v when handling delete event: %s", err, event)
+							glog.V(5).Infof("error %v when handling delete event: %s", err, event)
```

Comment:
Actually it was in 1.12: https://github.com/kubernetes/kubernetes/issues/69015#issuecomment-438223544

In 1.13 we didn't do that because moving to that new directory meant that no other files were created, thus log pollution wasn't really an issue. We probably should reduce the verbosity on master though.

Reply:
Looks like this PR is also reducing depth of search and that should help in reducing unwanted error logs to a large extent. I fear that we might miss some relevant error log by keeping verbosity 5.

Reply:
We ended up supporting the old directory even in 1.13 (it was readded in https://github.com/kubernetes/kubernetes/pull/71314). Because we exclude the `kubernetes.io/` log verbosity isn't a huge issue. So I would lean to leaving it as is like @vikaschoudhary16 suggested for ease of debugging.